scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -180,17 +184,22 @@ object EscapeHatch {
   }
 
   private object AnnotatedEscapes {
-    private val SuppressWarnings = "SuppressWarnings"
+    private val SuppressWarnings = classOf[SuppressWarnings].getSimpleName
```

Comment:
Is this necessary? I would avoid runtime reflection when possible even if that means hardcoding a string literal, the name is not going to change in the JDK anytime soon anyways.

Reply:
fair enough. Reverted